Change some host names and port numbers to localhost or 127.0.42.{1,2}
as appropriate.

===================================================================
--- frontend/config/environments/development_base.rb	(revision 2569)
+++ frontend/config/environments/development_base.rb	(working copy)
@@ -20,14 +20,14 @@
 
 BREAKPOINT_SERVER_PORT=42532
 
-SOURCE_HOST = "buildservice.suse.de"
-#SOURCE_HOST = "localhost"
+#SOURCE_HOST = "buildservice.suse.de"
+SOURCE_HOST = "localhost"
 
 #SOURCE_PORT = 5352
 SOURCE_PORT = 6362 #test backend
 
-RPM_HOST = "buildservice.suse.de"
-#RPM_HOST = "localhost"
+#RPM_HOST = "buildservice.suse.de"
+RPM_HOST = "localhost"
 
 #RPM_PORT = 5252
 RPM_PORT = 6262 #test backend
--- webclient/config/environments/development_base.rb	(revision 2569)
+++ webclient/config/environments/development_base.rb	(working copy)
@@ -18,8 +18,8 @@
 # Don't care if the mailer can't send
 config.action_mailer.raise_delivery_errors = false
 
-FRONTEND_HOST = "localhost"
-FRONTEND_PORT = 3001
+FRONTEND_HOST = "127.0.42.2"
+FRONTEND_PORT = 80
 #FRONTEND_PORT = 9001
 FRONTEND_PROTOCOL = 'http'
 
--- frontend/script/import	2007/11/15 14:12:48	1.1
+++ frontend/script/import	2007/11/15 14:14:38
@@ -28,7 +28,8 @@
 
     #change default_server to your backend
     #map.default_server :rest, "buildservice.suse.de:5352" #real backend
-    map.default_server :rest, "buildservice.suse.de:6362" #test backend
+    map.default_server :rest, "localhost:6362" #test backend
+
 
     map.connect :project, "bssql:///", :write_through => :false
     map.connect :package, "bssql:///", :write_through => :false
